specs/analytics_v1_pageviews.yaml

```diff
@@ -0,0 +1,291 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: Analytics Pageviews API
+  description: Analytics Pageviews API.
+  termsofservice: https://github.com/wikimedia/restbase#restbase
+  contact:
+    name: Analytics
+    email: analytics@lists.wikimedia.org
+    url: https://www.mediawiki.org/wiki/Analytics
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+x-default-params:
+  domain: analytics.wikimedia.org
+paths:
+  /{module:pageviews}/:
+    get:
+      tags:
+        - Pageviews data
+      description: >
+        List pageviews data sub-apis.
+
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Stable)
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: The queriable sub-items
+          schema:
+            $ref: '#/definitions/listing'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-monitor: false
+
+  /{module:pageviews}/per-article/{project}/{access}/{agent}/{article}/{granularity}/{start}/{end}:
+    get:
+      tags:
+        - Pageviews data
+        - per-article
+      description: >
+        List pageviews for a given article in a project, over a given time period.
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: project
+          in: path
+          description: Name of the project
+          type: string
+          required: true
+        - name: access
+          in: path
+          description: Type of access method [all-access|desktop|mobile-app|mobile-web]
+          type: string
```

Comment:
In case a string is actually a enum, you could add a [swagger enum](http://stackoverflow.com/questions/27603871/how-to-define-enum-in-swagger-io) to get a nice choice box in resulting docs.